[2024-05-14 00:29:25,697: INFO: main: welcome to our custom log]
[2024-05-14 00:32:36,210: INFO: main: welcome to our custom log]
[2024-05-15 13:51:45,686: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-15 13:51:45,697: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-15 13:51:45,697: INFO: common: created directory at: artifacts]
[2024-05-15 13:51:45,697: INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-15 13:51:45,709: INFO: 1579108944: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-15 13:51:57,204: INFO: 1579108944: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-15 14:50:22,518: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-15 14:50:22,530: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-15 14:50:22,533: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-15 14:50:22,533: INFO: common: created directory at: artifacts]
[2024-05-15 14:50:22,533: ERROR: main: 'function' object has no attribute 'source_URL']
Traceback (most recent call last):
  File "main.py", line 14, in <module>
    data_ingestion.main()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\pipelines\stage_01_data_ingestion.py", line 13, in main
    data_ingestion.download_file()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 31, in download_file
    raise e
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 19, in download_file
    dataset_url = self.config.source_URL
AttributeError: 'function' object has no attribute 'source_URL'
[2024-05-15 14:58:08,016: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-15 14:58:08,016: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-15 14:58:08,016: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-15 14:58:08,016: INFO: common: created directory at: artifacts]
[2024-05-15 14:58:08,016: ERROR: main: 'function' object has no attribute 'source_URL']
Traceback (most recent call last):
  File "main.py", line 14, in <module>
    data_ingestion.main()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\pipelines\stage_01_data_ingestion.py", line 13, in main
    data_ingestion.download_file()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 31, in download_file
    raise e
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 19, in download_file
    dataset_url = self.config.source_URL
AttributeError: 'function' object has no attribute 'source_URL'
[2024-05-15 15:10:46,892: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-15 15:10:46,894: ERROR: main: name 'configurationManager' is not defined]
Traceback (most recent call last):
  File "main.py", line 14, in <module>
    data_ingestion.main()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\pipelines\stage_01_data_ingestion.py", line 10, in main
    config=configurationManager()
NameError: name 'configurationManager' is not defined
[2024-05-15 15:12:44,676: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-15 15:12:44,676: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-15 15:12:44,676: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-15 15:12:44,676: INFO: common: created directory at: artifacts]
[2024-05-15 15:12:44,676: ERROR: main: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config']
Traceback (most recent call last):
  File "main.py", line 14, in <module>
    data_ingestion.main()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\pipelines\stage_01_data_ingestion.py", line 11, in main
    data_ingestion_config = config.get_data_ingestion_config
AttributeError: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config'
[2024-05-15 15:15:21,253: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-15 15:15:21,253: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-15 15:15:21,253: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-15 15:15:21,253: INFO: common: created directory at: artifacts]
[2024-05-15 15:15:21,266: ERROR: main: 'function' object has no attribute 'source_URL']
Traceback (most recent call last):
  File "main.py", line 14, in <module>
    data_ingestion.main()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\pipelines\stage_01_data_ingestion.py", line 13, in main
    data_ingestion.download_file()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 33, in download_file
    raise e
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 21, in download_file
    dataset_url = self.config.source_URL
AttributeError: 'function' object has no attribute 'source_URL'
[2024-05-15 15:16:19,536: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-15 15:16:19,536: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-15 15:16:19,536: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-15 15:16:19,536: INFO: common: created directory at: artifacts]
[2024-05-15 15:16:19,536: ERROR: main: 'function' object has no attribute 'source_URL']
Traceback (most recent call last):
  File "main.py", line 14, in <module>
    data_ingestion.main()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\pipelines\stage_01_data_ingestion.py", line 13, in main
    data_ingestion.download_file()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 33, in download_file
    raise e
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 21, in download_file
    dataset_url = self.config.source_URL
AttributeError: 'function' object has no attribute 'source_URL'
[2024-05-15 15:17:46,104: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-15 15:17:46,112: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-15 15:17:46,112: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-15 15:17:46,114: INFO: common: created directory at: artifacts]
[2024-05-15 15:17:46,115: ERROR: main: 'function' object has no attribute 'source_URL']
Traceback (most recent call last):
  File "main.py", line 14, in <module>
    data_ingestion.main()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\pipelines\stage_01_data_ingestion.py", line 13, in main
    data_ingestion.download_file()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 33, in download_file
    raise e
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 21, in download_file
    dataset_url = self.config.source_URL
AttributeError: 'function' object has no attribute 'source_URL'
[2024-05-15 15:19:19,062: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-15 15:19:19,062: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-15 15:19:19,062: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-15 15:19:19,072: INFO: common: created directory at: artifacts]
[2024-05-15 15:19:19,073: ERROR: main: 'function' object has no attribute 'source_URL']
Traceback (most recent call last):
  File "main.py", line 14, in <module>
    data_ingestion.main()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\pipelines\stage_01_data_ingestion.py", line 13, in main
    data_ingestion.download_file()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 33, in download_file
    raise e
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 21, in download_file
    dataset_url = self.config.source_URL
AttributeError: 'function' object has no attribute 'source_URL'
[2024-05-15 15:22:16,977: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-15 15:22:16,980: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-15 15:22:16,982: INFO: common: created directory at: artifacts]
[2024-05-15 15:22:16,984: INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-15 15:22:16,986: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-15 15:22:28,206: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-15 15:27:57,105: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-15 15:27:57,105: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-15 15:27:57,118: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-15 15:27:57,118: INFO: common: created directory at: artifacts]
[2024-05-15 15:27:57,118: ERROR: main: 'function' object has no attribute 'source_URL']
Traceback (most recent call last):
  File "main.py", line 14, in <module>
    data_ingestion.main()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\pipelines\stage_01_data_ingestion.py", line 13, in main
    data_ingestion.download_file()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 33, in download_file
    raise e
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 21, in download_file
    dataset_url = self.config.source_URL
AttributeError: 'function' object has no attribute 'source_URL'
[2024-05-15 15:29:02,864: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-15 15:29:02,874: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-15 15:29:02,875: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-15 15:29:02,876: INFO: common: created directory at: artifacts]
[2024-05-15 15:29:02,876: ERROR: main: 'function' object has no attribute 'source_URL']
Traceback (most recent call last):
  File "main.py", line 14, in <module>
    data_ingestion.main()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\pipelines\stage_01_data_ingestion.py", line 13, in main
    data_ingestion.download_file()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 33, in download_file
    raise e
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 21, in download_file
    dataset_url = self.config.source_URL
AttributeError: 'function' object has no attribute 'source_URL'
[2024-05-15 15:33:00,745: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-15 15:33:00,756: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-15 15:33:00,757: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-15 15:33:00,757: INFO: common: created directory at: artifacts]
[2024-05-15 15:33:00,758: ERROR: main: 'function' object has no attribute 'source_URL']
Traceback (most recent call last):
  File "main.py", line 11, in <module>
    data_ingestion.main()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\pipelines\stage_01_data_ingestion.py", line 13, in main
    data_ingestion.download_file()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 33, in download_file
    raise e
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 21, in download_file
    dataset_url = self.config.source_URL
AttributeError: 'function' object has no attribute 'source_URL'
[2024-05-15 15:42:19,435: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-15 15:42:19,435: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-15 15:42:19,435: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-15 15:42:19,435: INFO: common: created directory at: artifacts]
[2024-05-15 15:42:19,435: ERROR: main: 'function' object has no attribute 'source_URL']
Traceback (most recent call last):
  File "main.py", line 11, in <module>
    data_ingestion.main()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\pipelines\stage_01_data_ingestion.py", line 13, in main
    data_ingestion.download_file()
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 33, in download_file
    raise e
  File "c:\users\asus\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 21, in download_file
    dataset_url = self.config.source_URL
AttributeError: 'function' object has no attribute 'source_URL'
[2024-05-15 15:46:06,485: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-15 15:46:06,487: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-15 15:46:06,488: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-15 15:46:06,488: INFO: common: created directory at: artifacts]
[2024-05-15 15:46:06,488: INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-15 15:46:06,490: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-15 15:46:17,607: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-15 15:46:18,748: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-05-15 15:47:31,909: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-15 15:47:31,915: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-15 15:47:31,915: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-15 15:47:31,915: INFO: common: created directory at: artifacts]
[2024-05-15 15:47:31,915: INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-15 15:47:31,915: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-15 15:47:44,474: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-15 15:47:45,698: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-05-15 16:03:34,011: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-15 16:03:34,025: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-15 16:03:34,025: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-15 16:03:34,025: INFO: common: created directory at: artifacts]
[2024-05-15 16:03:34,025: INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-15 16:03:34,028: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-15 16:03:45,574: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-15 16:03:46,599: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-05-15 17:41:24,094: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-15 17:41:24,110: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-15 17:41:24,112: INFO: common: created directory at: artifacts]
[2024-05-15 17:41:24,114: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-05-15 17:41:35,976: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-15 23:02:58,716: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-05-15 23:02:58,720: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-15 23:02:58,720: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-15 23:02:58,720: INFO: common: created directory at: artifacts]
[2024-05-15 23:02:58,725: INFO: common: created directory at: artifacts/data_ingestion]
[2024-05-15 23:02:58,726: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-15 23:03:12,329: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-05-15 23:03:13,782: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-05-15 23:03:13,783: INFO: main: *******************]
[2024-05-15 23:03:13,784: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-05-15 23:03:13,785: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-05-15 23:03:13,785: INFO: common: yaml file: params.yaml loaded successfully]
[2024-05-15 23:03:13,785: INFO: common: created directory at: artifacts]
[2024-05-15 23:03:13,793: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-05-15 23:03:14,472: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-05-15 23:03:14,787: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
